generators/git/index.js

```diff
@@ -26,10 +26,6 @@ module.exports = generators.Base.extend({
 
   end: function () {
     this.spawnCommandSync('git', ['init']);
-
-    if (!this.options.repositoryPath) {
-      var repoSSH = 'git@github.com:' + this.options.repositoryPath + '.git';
-      this.spawnCommandSync('git', ['remote', 'add', 'origin', repoSSH]);
-    }
+    this.spawnCommandSync('git', ['remote', 'add', 'origin', this.options.repositoryPath]);
```

Comment:
I don't see where we garantee repositoryPath to be a full url. From reading the code is seems to me it is still manually setup as a `name/repo` format.